ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,29 +4,5 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes:
-  - enabled: 1
-    path: Assets/Scenes/levels/gameOver.unity
-    guid: e06d85b9cc01b5147868064520f4ea65
-  - enabled: 1
-    path: Assets/Scenes/levels/goodEnding.unity
-    guid: fc851e67a87429048bd8155baceb9115
-  - enabled: 1
-    path: Assets/Scenes/levels/neutralEnding.unity
-    guid: 25644727d239a3c4bb5a63b79433fb01
-  - enabled: 1
-    path: Assets/Scenes/levels (don't edit)/menu.unity
-    guid: d69a152602cdb60468fa0123e1b6e194
-  - enabled: 1
-    path: Assets/Scenes/levels (don't edit)/morning.unity
-    guid: 82502a4a3e573a777ae0ddfaaf136cb0
-  - enabled: 1
-    path: Assets/Scenes/levels (don't edit)/night.unity
-    guid: 1850203cc0f87bb1ca8cc69cb16f2bf2
-  - enabled: 1
-    path: Assets/Scenes/levels (don't edit)/afternoon.unity
-    guid: 6ea732993ba09af0698b4197a6062131
-  - enabled: 1
-    path: Assets/Scenes/levels (don't edit)/evening.unity
-    guid: 0ffc1cd5a381384ca9c8baa399eb4f69
+  m_Scenes: []
```

Comment:
Why build settings are removed?

Reply:
I am not sure I think this happened last time as well... when I open my unity it automatically says i did something to the editor setting.

Reply:
how should i fix that? I didn't get any conflicts when i try pushing and pull request though